index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="header">Hello Microverse</h1>
+    <h1 class="header">is one</h1>
+    <h1 class="header">of clear and understandable project</h1>
```

Comment:
Please ensure that you use only one h1 in your entire HTML document, using only a single <h1> element in an HTML document neglects the benefits of proper structure, accessibility, SEO, and visual consistency that multiple heading levels provide. It is recommended to use the appropriate heading levels to organize and present your content effectively. 🙂 1+1: